octobot_trading/enums.py

```diff
@@ -217,6 +223,7 @@ class ExchangeOrderCCXTColumns(enum.Enum):
     REMAINING = "remaining"
     STATUS = "status"
     FEE = "fee"
+    FEES = "fees"
```

Comment:
As my first comment, I don't think we should have both fee and fees

Reply:
yeah its there but not documented, same for the stopPrice
this is binance spot, bybit looks the same 
![image](https://user-images.githubusercontent.com/32987678/207023884-ea215357-be83-48cf-9bab-8c6f8a5fcb6f.png)

so I think we should add both stopPrice and fees to the default ccxt parser

Reply:
Sure, but if there is not "fee" anymore in ccxt response we may remove it. Our point is that we shouldn't have both "fees" and "fee"

Reply:
CCXT response is with both, that's why I added both, if you want only one, I'm not sure which one to keep

Reply:
Can you check in the ccxt doc why there is two "fees" please?